cognition_objects/message.py

```diff
@@ -189,3 +195,133 @@ def delete(project_id: str, message_id: str, with_commit: bool = True) -> None:
         CognitionMessage.id == message_id,
     ).delete()
     general.flush_or_commit(with_commit)
+
+
+def get_messages_per_conversation_query(project_id: str) -> str:
+    project_id = prevent_sql_injection(project_id, isinstance(project_id, str))
+    return f"""
+    SELECT 
+        conversation_id,
+        COUNT(*) messages,
+        c."header"
+    FROM cognition.message as m
+        inner join cognition.conversation c on c.id = m.conversation_id and c.project_id = m.project_id 
+    WHERE m.project_id = '{project_id}'
+    GROUP BY conversation_id, c."header" 
+    ORDER BY conversation_id, c."header"
```

Comment:
```suggestion
  SELECT 
    conversation_id,
    c."header",
    COUNT(*) messages
  FROM cognition.message m
  INNER JOIN cognition.conversation c
	  ON c.project_id = m.project_id AND c.id = m.conversation_id 
  WHERE m.project_id = '{project_id}'
  GROUP BY 1,2
  ORDER BY 1,2
```

No logic changes just styling to macht our guide & group & order without names to write less